homeassistant/components/upb/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "upb",
   "name": "Universal Powerline Bus (UPB)",
   "documentation": "https://www.home-assistant.io/integrations/upb",
-  "requirements": ["upb_lib==0.4.11"],
+  "requirements": ["upb_lib==0.4.12"],
```

Comment:
The library source doesn't seem to have been changed?

https://github.com/gwww/upb-lib/blob/3e5a64cde58b624d8a8da4415d96de4e1c15e613/pyproject.toml#L3

Reply:
My mistake. Fixed.